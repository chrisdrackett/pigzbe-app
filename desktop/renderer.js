import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
