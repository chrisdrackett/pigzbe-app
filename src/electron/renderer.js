import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
