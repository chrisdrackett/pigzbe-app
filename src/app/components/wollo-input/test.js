import React from 'react';
import renderer from 'react-test-renderer';
import WolloInput from './';

const props = {
    currency: 'GBP',
    exchange: 0.091956,
    onChangeAmount: () => {}
};

describe('WolloInput', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<WolloInput {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
