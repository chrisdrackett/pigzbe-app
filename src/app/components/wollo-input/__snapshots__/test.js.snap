// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WolloInput renders correctly 1`] = `
Array [
  <View
    style={
      Object {
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <View
      style={
        Object {
          "marginBottom": 10,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignSelf": "stretch",
              "borderColor": "rgb(127, 153, 188)",
              "borderRadius": 22.5,
              "borderWidth": 1,
              "paddingLeft": 22,
              "paddingRight": 22,
            },
            Object {
              "borderRadius": 25,
              "height": 50,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "rgb(0, 50, 120)",
              "fontFamily": "Poppins",
              "fontSize": 14,
              "fontWeight": "bold",
              "left": 22,
              "opacity": 0.6,
              "position": "absolute",
              "top": 14,
            }
          }
        >
          0.00 Wollo
        </Text>
        <TextInput
          allowFontScaling={true}
          autoCapitalize="sentences"
          autoCorrect={true}
          blurOnSubmit={true}
          editable={true}
          keyboardType="default"
          multiline={false}
          numberOfLines={1}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          placeholder=""
          placeholderTextColor="rgb(127, 153, 188)"
          returnKeyType="done"
          style={
            Array [
              Object {
                "color": "rgb(0, 50, 120)",
                "flex": 1,
                "fontFamily": "Poppins",
                "fontSize": 14,
                "fontWeight": "bold",
                "lineHeight": 21,
                "textAlignVertical": "top",
              },
              Object {
                "paddingTop": 10,
                "width": "100%",
              },
              Object {
                "width": 560,
              },
            ]
          }
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "rgb(127, 153, 188)",
          "borderRadius": 22.5,
          "height": 45,
          "opacity": 1,
          "position": "relative",
          "width": 80,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "rgb(0, 50, 120)",
            "borderRadius": 20.5,
            "height": 41,
            "left": 2,
            "position": "absolute",
            "shadowColor": "rgb(0, 0, 0)",
            "shadowOffset": Object {
              "height": 2,
              "width": 1,
            },
            "shadowOpacity": 0.3,
            "shadowRadius": 2,
            "top": 2,
            "width": 41,
          }
        }
      />
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/currency-toggle/images/wollo.png",
          }
        }
        style={
          Object {
            "height": 13,
            "left": 14,
            "position": "absolute",
            "top": 16,
            "width": 16,
          }
        }
      />
      <View
        style={
          Object {
            "position": "absolute",
            "right": 17,
            "top": 13,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "rgb(255, 255, 255)",
              "fontSize": 16,
              "fontWeight": "bold",
            }
          }
        >
          £
        </Text>
      </View>
    </View>
  </View>,
  <Text
    style={
      Object {
        "color": "rgb(127, 153, 188)",
        "fontSize": 16,
        "fontStyle": "italic",
        "marginTop": 5,
        "textAlign": "left",
      }
    }
  >
    Estimate: 
    £0
  </Text>,
]
`;
